docs/pages/video.js

```diff
@@ -366,7 +365,17 @@ function Example () {
 export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
   const generatedDocGen = await docgen({ componentName: 'Video' });
 
+  generatedDocGen.props.ref = {
+    required: false,
+    defaultValue: null,
+    flowType: {
+      name: 'Ref',
+      raw: 'Ref<typeof Video>',
+    },
+    description: 'Ref on the Gestalt Video component.',
+  };
+
   return {
-    props: { generatedDocGen: deepCloneReplacingUndefined(generatedDocGen) },
```

Comment:
Just making sure you double-check that the page loads with no errors. I don't recall the exact error this was necessary for, but it was something about undefined values in the JSON